### DISCUSSION TITLE
Can the rotation action be rotated clockwise?

### DISCUSSION THREAD

**Discussion by taara67** [iRobotEducation/create3_docs#490] [Questions] [answered]
### How are you connecting to your Create 3?

USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

jetson Orin nano Ubuntu20.04

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

G.5.3

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

Hello, I am thinking of implementing a function to follow people using 3DLiDAR.
A simple function that rotates and moves forward in the direction of the human body could be implemented using rclpy, but since we enter a value from 0 to 2π in the rotatWe eaction argument, in a situation that Create3 would be enough to turn 1π/8 to the right, it turn 15π/8 to the left.
I would like to solve this problem.

**Comment by brianabouchard** [comment, marked as answer]
Hello! I'm happy to help. Just to confirm, are you using the pre-built Create 3 action [RotateAngle.action](https://github.com/iRobotEducation/irobot_create_msgs/blob/rolling/action/RotateAngle.action)? 

Now, for the problem itself, is the issue that you want the Create 3 to rotate in both directions? What happens when you enter a negative value for the rotation angle?

**Comment by taara67** [comment]
oh
I had overlooked it. It seems possible to input negative angles as well.
thank you
 
by the way,
Which site would you recommend to find sample code for various systems that use create_msgs?

I would like to see various codes to learn rclpy.

**Reply by brianabouchard** [reply]
Excellent! 

We're in the process of adding additional python examples to the official [Create 3 Examples Repository](https://github.com/iRobotEducation/create3_examples). In the meantime, you can check out additional examples using rclpy with the Create 3 robot on this [student produced repository](https://github.com/tuftsceeo/Tufts_Create3_Examples).
